HTML/Miscellaneous.tmPreferences

```diff
@@ -9,29 +9,25 @@
 	<dict>
 		<key>decreaseIndentPattern</key>
 		<string><![CDATA[(?x)
-			^\s*
-			<\?(php)?\s+(else(if)?|end(if|for(each)?|while))
-		]]></string>
-		<key>batchDecreaseIndentPattern</key>
-		<string><![CDATA[(?x)
-			^\s*
-			(</(?!html)
-			  [A-Za-z0-9-]+\b[^>]*>
-			|-->
-			|<\?(php)?\s+(else(if)?|end(if|for(each)?|while))
-			|\}
+			^\s*                                                                              # the beginning of the line followed by any amount of whitespace
+			(
+			    <\?(php)?\s+(else(if)?|end(if|for(each)?|while))                              # ending php control keywords
+			|   </(?!html)[A-Za-z0-9-]+\b[^>]*>                                               # any valid close tag except html
+			|   -->                                                                           # closing comment punctuation
+			|   \}                                                                            # a closing brace
 			)
 		]]></string>
 		<key>increaseIndentPattern</key>
-		<string><![CDATA[^\s*<\?php\s*\b(if|else|elseif|for|foreach|while)\b.*:(?!.*end\1)]]></string>
-		<key>batchIncreaseIndentPattern</key>
 		<string><![CDATA[(?x)
-			^\s*
-			<(?!\?|area|base|br|col|frame|hr|html|img|input|link|meta|param|[^>]*/>)
-			  ([A-Za-z0-9-]+)(?=\s|>)\b[^>]*>(?!.*</\1>)
-			|<!--(?!.*-->)
-			|<\?php.+?\b(if|else(?:if)?|for(?:each)?|while)\b.*:(?!.*end\1)
-			|\{[^}"']*$
+			^\s*                                                                              # the beginning of the line followed by any amount of whitespace
+			(
+			    <\?(php)?\s*\b(?<php_control_word>if|else|elseif|for|foreach|while)\b.*:      # php control keywords
+			    (?!.*end\g<php_control_word>)                                                 # - that don't end themselves on the same line
```

Comment:
I've pushed a new commit to correct this. It seems that ST's Find panel works differently compared to the indentation rules (even though I thought they use the same Boost engine), so trying to use a conditional in these rules just causes ST to crash (https://github.com/SublimeTextIssues/Core/issues/1383) so I've used a slightly different approach. Maybe we can revise it after the next build is released (which has a fix for this crash bug), when we can see what the error actually is, but I'm happy with the approach I've used for now.